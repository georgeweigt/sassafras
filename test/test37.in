title "Anova Procedure Demo"

data;
input Block $ A $ B $ Response
datalines
1 4 2 40.0
1 4 1 39.5
1 1 2 37.9
1 1 1 35.4
1 2 1 36.7
1 2 2 38.2
1 3 2 36.4
1 3 1 34.8
2 2 1 42.7
2 2 2 41.6
2 1 2 40.3
2 1 1 41.6
2 4 1 44.5
2 4 2 47.6
2 3 1 43.6
2 3 2 42.8
;

proc anova
model Response = Block A Block*A B A*B
means Block

