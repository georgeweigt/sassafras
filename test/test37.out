                              Anova Procedure Demo

                              Analysis of Variance

    Source     DF     Sum of Squares     Mean Square     F Value     Pr > F
    Model      11       182.02000000     16.54727273        7.85     0.0306
    Error       4         8.43000000      2.10750000                       
    Total      15       190.45000000                                       

             R-Square     Coeff Var     Root MSE     RESPONSE Mean
             0.955736      3.609007     1.451723         40.225000

    Source      DF         Anova SS      Mean Square     F Value     Pr > F
    BLOCK        1     131.10250000     131.10250000       62.21     0.0014
    A            3      40.19000000      13.39666667        6.36     0.0530
    BLOCK*A      3       6.92750000       2.30916667        1.10     0.4476
    B            1       2.25000000       2.25000000        1.07     0.3599
    A*B          3       1.55000000       0.51666667        0.25     0.8612

                                 Mean Response

          BLOCK     N     Mean RESPONSE     95% CI MIN     95% CI MAX
          1         8         37.362500      35.937457      38.787543
          2         8         43.087500      41.662457      44.512543

